blockchain/config.go

```diff
@@ -42,43 +42,43 @@ var (
-	Timeout StrDuration `envconfig:"evm_transaction_timeout" default:"2m" toml:"evm_transaction_timeout" json:"evm_transaction_timeout"`
+	Timeout StrDuration `toml:"evm_transaction_timeout" json:"evm_transaction_timeout"`
 	// How many block confirmations to wait to confirm on-chain events
-	MinimumConfirmations int `envconfig:"evm_minimum_confirmations" default:"1" toml:"evm_minimum_confirmations" json:"evm_minimum_confirmations"`
+	MinimumConfirmations int `toml:"evm_minimum_confirmations" json:"evm_minimum_confirmations"`
 	// How much WEI to add to gas estimations for sending transactions
-	GasEstimationBuffer uint64 `envconfig:"evm_gas_estimation_buffer" default:"1000" toml:"evm_gas_estimation_buffer" json:"evm_gas_estimation_buffer"`
+	GasEstimationBuffer uint64 `toml:"evm_gas_estimation_buffer" json:"evm_gas_estimation_buffer"`
 	// ClientImplementation is the blockchain client to use when interacting with the test chain
-	ClientImplementation ClientImplementation `envconfig:"client_implementation" default:"Ethereum" toml:"client_implementation" json:"client_implementation"`
+	ClientImplementation ClientImplementation `toml:"client_implementation" json:"client_implementation"`
 	// SupportsEIP1559 indicates if the client should try to use EIP1559 style gas and transactions
-	SupportsEIP1559 bool `envconfig:"evm_supports_eip1559" default:"false" toml:"evm_supports_eip1559" json:"evm_supports_eip1559"`
+	SupportsEIP1559 bool `toml:"evm_supports_eip1559" json:"evm_supports_eip1559"`
 
 	// Default gaslimit to use when sending transactions. If set this will override the transactionOptions gaslimit in case the
 	// transactionOptions gaslimit is lesser than the defaultGasLimit.
-	DefaultGasLimit uint64 `envconfig:"evm_default_gas_limit" default:"500000" toml:"evm_default_gas_limit" json:"evm_default_gas_limit"`
+	DefaultGasLimit uint64 `toml:"evm_default_gas_limit" json:"evm_default_gas_limit"`
 	// Few chains use finality tags to mark blocks as finalized. This is used to determine if the chain uses finality tags.
-	FinalityTag bool `envconfig:"evm_finality_tag" default:"false" toml:"evm_finality_tag" json:"evm_finality_tag"`
+	FinalityTag bool `toml:"evm_finality_tag" json:"evm_finality_tag"`
 	// If the chain does not use finality tags, this is used to determine how many blocks to wait for before considering a block finalized.
-	FinalityDepth uint64 `envconfig:"evm_finality_depth" default:"50" toml:"evm_finality_depth" json:"evm_finality_depth"`
+	FinalityDepth uint64 `toml:"evm_finality_depth" json:"evm_finality_depth"`
 
 	// TimeToReachFinality is the time it takes for a block to be considered final. This is used to determine how long to wait for a block to be considered final.
-	TimeToReachFinality StrDuration `envconfig:"evm_time_to_reach_finality" default:"0s" toml:"evm_time_to_reach_finality" json:"evm_time_to_reach_finality"`
+	TimeToReachFinality StrDuration `toml:"evm_time_to_reach_finality" json:"evm_time_to_reach_finality"`
```

Comment:
I'm not sure we should be deleting defaults, even though it might get in a way of TOML config... Although I think it boils down to this question: are known networks in the "dictionary" defining all parameters or are they using any defaults?

Reply:
@kalverra any idea if we are using the defaults?

Reply:
I will bring defaults back for now